var/spack/repos/builtin/packages/turbine/package.py

```diff
@@ -38,8 +38,8 @@ class Turbine(AutotoolsPackage):
     depends_on('zsh')
```

Comment:
Is this a runtime dependency?  In that case it would be: `depends_on('zsh', type='run')`